### PR TITLE
Feature: Job API exposing tag_list

### PR DESCRIPTION
**Pull request by burdandrei**
https://gitlab.com/gitlab-org/gitlab/-/merge_requests/44859

### CODE REVIEW

File: jobs.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ type Job struct {
 	ArtifactsExpireAt *time.Time `json:"artifacts_expire_at"`
 	ID                int        `json:"id"`
 	Name              string     `json:"name"`
+	TagList           []string   `json:"tag_list"`
```
Comment (svanharmelen) [inline jobs.go:48]: Please place this between `ArtifactsExpireAt` and `ID`, following the ordering in the docs.

Comment (burdandrei) [reply jobs.go]: Done!

**Review by svanharmelen — APPROVED** [review]
Thanks @burdandrei! LGTM!

**Comment by burdandrei** [comment]
The pleasure is mine @svanharmelen.
will we see 0.45 soon?

**Comment by svanharmelen** [comment]
Done 👍🏻

**Pull request merged**